src/app/interceptors/custom-header.interceptor.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ *    Copyright 2020 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { versions } from '../footer/versions';
+
+const uiVersion = versions.tag.match(/\d+\.\d+\.\d+/)[0];
```

Comment:
Why are you doing this instead of just using `versions.tag` as is? What's "beyond" what you're capturing, and why don't you want to include it? Not saying it's wrong; just asking for clarification.

Reply:
An example of versions.tag is ``2.5.1-rc.1-29-g06e87643`` so I just captured ``2.5.1`` as the version based on your previous comment about the current UI version being 2.5.1. I wasn't sure if the info after that was needed. Should I keep the whole thing?

Reply:
Hmm, not sure... Does it cause issues for metrics if you use the long name? For staging, I think we would at least want the `-rc.1` portion of the name.

Reply:
I can use regular expressions when I create the metric filters, so shouldn't cause any issues for metrics if I use the long name since I can still get what I need. I'll use the full name then